docs/contributor/network/multus.md

```diff
@@ -0,0 +1,204 @@
+# Multus in MicroShift
+
+MicroShift 4.16 includes an optional Multus CNI.
+It can be used to attach additional interfaces to Pods.
+
+## How to install
+
+Manifests to deploy Multus CNI are part of `microshift-multus` RPM.
+Depending on the timeline it can be installed from OpenShift mirror or built locally.
+Specific instructions depend on source of the RPM.
+After downloading or building the RPM you can install it by running command:
+```sh
+$ sudo dnf install microshift-multus*.rpm
+```
+If the RPM is available via enabled RHOCP or OpenShift mirror was added to the system repositories it can be installed using:
+```sh
+$ sudo dnf install microshift-multus
+```
+
+## Supported plugins on MicroShift
+
+> See section "More information" for external resources.
+
+CNI:
+- Bridge
+- macvlan
+- ipvlan
+
+> Fun fact: ipvlan and macvlan are not related to the VLAN.
+
+IPAM (IP Address Management):
+- host-local
+- static
+- dhcp
+
+## How to use
+
+> Following is a simple example based on one of the CI tests.
+
+It is assumed that the MicroShift cluster is already up and running (including Multus).
+
+```
+$ oc get pods -n openshift-multus 
+NAME                READY   STATUS    RESTARTS   AGE
+dhcp-daemon-dfbzw   1/1     Running   0          5h
+multus-rz8xc        1/1     Running   0          5h
+```
+
+First step is creation of NetworkAttachmentDefinition.
+It is basically a wrapper of raw CNI config in a form of Kubernetes' custom resource.
+
+```sh
+$ oc apply -f ./test/assets/multus/bridge-nad.yaml
+```
+
+Here's content of the file:
```

Comment:
Should we link the file in the repo, rather than dumping its contents? This may get outdated.

Reply:
I'm not sure if file getting outdated is really necessary because it serves as an example that is later explained. Also, needing to open two pages (content + explanation) would be cumbersome IMHO.